docs/mobile-apps/automated-testing/appium/appium-2-deque-drivers.md

```diff
@@ -0,0 +1,174 @@
+has been deprecated, Sauce Labs will continue hosting it until January 31st 2025, allowing users time to migrate to the new drivers.
+- ****Powered by Appium :**** This version is built on version [2.12.1](https://github.com/appium/appium/releases/tag/appium%402.12.1) of appium, 
+ensuring the latest Appium features and compatibility enhancements.
+
+## How to Use `appium2-deque-accessibility`
+
+Follow these steps to set up and execute accessibility tests using the `appium2-deque-accessibility` image, leveraging Deque's Appium drivers for Android and iOS.
+
+1. Set Up Your Appium Capabilities
+
+<Tabs
+groupId="appium2-deque-accessibility"
+defaultValue="AxeUIAutomator2"
+values={[
+{label: 'AxeUIAutomator2', value: 'AxeUIAutomator2'},
+{label: 'AxeXCUITEST', value: 'AxeXCUITEST'},
+]}>
+
+<TabItem value="AxeUIAutomator2">
+
+<!-- prettier-ignore -->
+```js
+const capabilities = {
+  platformName: 'Android',
+  'appium:platformVersion': '1[0-9]',
+  'appium:automationName': 'AxeUIAutomator2', // New Automation name goes here
+  'appium:app': 'storage:filename=XXXXXXXXXx',
+  'appium:appPackage': 'xxxxxxxxxx',
+  'appium:autoGrantPermissions': true,
+  'appium:autoAcceptAlerts': true,
+  'appium:newCommandTimeout': 90,
+  'sauce:options': {
+    name: "Android Driver - Testing appium2-deque-accessibility",
+    appiumVersion: 'appium2-deque-accessibility',
+  },
+}
+```
+</TabItem>
+<TabItem value="AxeXCUITEST">
+
+<!-- prettier-ignore -->
+```js
+const capabilities = {
+  platformName: 'iOS',
+  'appium:platformVersion': '^1(7|8).*$', // The iOS driver will work only on devices running iOS 17 and above
+  'appium:automationName': 'AxeXCUITEST', // New Automation name goes here
+  'appium:app': 'storage:filename=XXXXXXXXXx',
+  'appium:appPackage': 'xxxxxxxxxx',
+  'appium:autoGrantPermissions': true,
+  'appium:autoAcceptAlerts': true,
+  'appium:newCommandTimeout': 90,
+  'sauce:options': {
+    name: "iOS Driver - Testing appium2-deque-accessibility",
+    resigningEnabled: true,
+    appiumVersion: 'appium2-deque-accessibility',
+  },
+}
+```
+</TabItem>
+
+</Tabs>
+
+2. Obtain Your Deque API Key
+Log in to your [axe Deque account setting page](https://axe.deque.com/settings) and retrieve your API key. 
+This key is required to authenticate and initiate accessibility scans during your tests.
+
+3. Prepare Your Testing Script
+Once you have your API key, include it in your test script to perform a scan using the `mobile: axeScan` command. Below is an example script:
+
+```js
+describe("Test appium2-deque-accessibility - Using Drivers Android/iOS", () => {
+    it('test appium2-deque-accessibility', async () => {
+        const apiKey = process.env.DEQUE_API_KEY;
+        const settings = { apiKey: apiKey }
+
+        await driver.execute('mobile: axeScan', settings);
+    });
+});
+```
+## Migrating from the Deprecated `axeDevToolsMobile` Plugin to Deque Drivers
```

Comment:
I think its enough to mention the versions in this header. 

Migration from axe-dequeue-image:xyz to ...

Reply:
done

Reply:
I dont see that this is done

Reply:
I don't understand your comment here. do you want me to update the title ? or the description here ? because for me it's clear the way it is